.size-limit.js

```diff
@@ -40,7 +40,7 @@ module.exports = [
     path: 'packages/browser/build/npm/esm/index.js',
     import: createImport('init', 'browserTracingIntegration'),
     gzip: true,
-    limit: '36 KB',
+    limit: '36.5 KB',
```

Comment:
we're just ~15 bytes over the previous check but I figured I'd give the limit a slightly bigger bump as we usually do